apps/test/unit/code-studio/pd/workshop_enrollment/enroll_formTest.js

```diff
@@ -438,20 +387,10 @@ describe('Enroll Form', () => {
       ...pick(extraParams, ['role', 'grades_teaching']),
     };
     beforeEach(() => {
-      enrollForm = shallow(
-        <EnrollForm
-          user_id={props.user_id}
-          workshop_id={props.workshop_id}
-          workshop_course="CS Fundamentals"
-          first_name={props.first_name}
-          email={props.email}
-          previous_courses={props.previous_courses}
-          onSubmissionComplete={props.onSubmissionComplete}
-        />
-      );
+      enrollForm = renderDefault();
     });
 
-    it('submit other school_info fields when no school_id', () => {
+    it.skip('submit other school_info fields when no school_id', () => {
```

Comment:
I'm skipping school_info tests for now since they'll need to be updated when the new school info component is added in a fast follow pr for the same jira ticket

same on line 418 and 480 below